python/paddle/tensor/math.py

```diff
@@ -172,27 +172,22 @@ def pow(x, y, name=None):
             x = paddle.to_tensor([1, 2, 3])
             y = 2
             res = paddle.pow(x, y)
-            print(res.numpy()) # [1 4 9]
+            print(res) # [1 4 9]
             
             # example 2: y is a Tensor
             y = paddle.full(shape=[1], fill_value=2, dtype='float32')
             res = paddle.pow(x, y)
-            print(res.numpy()) # [1 4 9]
+            print(res) # [1 4 9]
 
     """
     # in dynamic graph mode
     if in_dygraph_mode():
         if isinstance(y, (int, float)):
             return core.ops.pow(x, 'factor', y)
         elif isinstance(y, (paddle.Tensor, Variable)):
-
             if x.dtype != y.dtype:
                 y = cast(y, dtype='float64')
                 x = cast(x, dtype='float64')
```

Comment:
The following code affects this version of paddle.pow api.
This block of code is meant to Align paddlepaddle pow api's functionality and behavior with numpy.power api, but it slowdowns the speed of this api call by 90%.